plutarch-extra/Plutarch/Extra/Map.hs

```diff
@@ -250,6 +254,16 @@ psortedMapFromFoldable = foldl' go pempty
       Term s' (PMap 'Sorted k v)
     go acc (key, val) = pinsert # key # val # acc
 
+instance (PIsData k, PIsData v, POrd k) => IsList (Term s (PMap 'Unsorted k v)) where
```

Comment:
Orphan instances are very bad. Just move the relevant functions into the defining module.

Reply:
to the module that defines PMap

Reply:
Ah, okay. Thank you.